ci/official/envs/release_libtensorflow_linux_x86_cpu

```diff
@@ -0,0 +1,7 @@
+source ci/official/envs/ci_nightly_uploads
+TFCI_BAZEL_COMMON_ARGS="--config release_cpu_linux --config tf_public_cache_push --config resultstore --repo_env=TF_PYTHON_VERSION=$TFCI_PYTHON_VERSION"
```

Comment:
I thought we didn't use the public_cache on release?

Reply:
You're right, thanks. I messed this up. Resolved in latest version.